backend_new/types/src/backend-swagger.ts

```diff
@@ -1158,3 +1314,18 @@ export enum UnitTypeEnum {
   'SRO' = 'SRO',
   'fiveBdrm' = 'fiveBdrm',
 }
+
+export enum UnitAccessibilityPriorityTypeEnum {
+  'mobility' = 'mobility',
+  'mobilityAndHearing' = 'mobilityAndHearing',
+  'hearing' = 'hearing',
+  'visual' = 'visual',
+  'hearingAndVisual' = 'hearingAndVisual',
+  'mobilityAndVisual' = 'mobilityAndVisual',
+  'mobilityHearingAndVisual' = 'mobilityHearingAndVisual',
+}
+
+export enum UnitRentTypeEnum {
+  'fixed' = 'fixed',
+  'percentageOfIncome' = 'percentageOfIncome',
+}
```

Comment:
It looks like we weren't consistent with this with enums already, but I think enums should be ALL_CAPS or PascalCase/UpperCamelCase and are the single quotes necessary on the left side? Following the convention shown here: https://www.typescriptlang.org/docs/handbook/enums.html#string-enums (ignoring the all caps on the right side), UnitRentTypeEnum would be:
```
export enum UnitRentTypeEnum {
    Fixed = 'fixed'
    PercentageOfIncome = 'percentageOfIncome'
}

Reply:
after offline convo with sean we're going to come to a consensus and address this in the standardization ticket 

https://www.prisma.io/docs/concepts/components/prisma-schema/data-model#defining-enums prisma doesn't allow for enums with a difference between its key and value so whatever we choose is what we end up choosing